django/cantusdb_project/main_app/tests/test_views.py

```diff
@@ -2917,6 +2934,22 @@ def test_dd_column(self):
         expected_html_substring: str = f'<a href="https://differentiaedatabase.ca/differentia/{diff_id}" target="_blank">'
         self.assertIn(expected_html_substring, html)
 
+    def test_redirect_with_source_parameter(self):
+        cantus_segment = make_fake_segment(id=4063)
+        source = make_fake_source(segment=cantus_segment)
+        source_id = source.id
+
+        url = reverse("redirect-chant-index")
+        response = self.client.get(f"{url}?source={source_id}")
+        self.assertRedirects(response, reverse("chant-inventory", args=[source_id]))
+
+    def test_redirect_without_source_parameter(self):
+        url = reverse("redirect-chant-index")
```

Comment:
The main place this needs to be changed is in `urls.py`, but this view should be `"redirect-chant-inventory", shouldn't it?

More generally, shouldn't this view now be called `SourceInventoryView`, and live in `views/source.py`? (Since/if this is the case, we maybe still want the redirect view to be "redirect-chant-index" or "-inventory", since we're redirecting from the old way the view was set up, with `/chant/` in the URL path...)